README.md

```diff
@@ -1 +1,2568 @@
  Authors Haven - React Project
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
```

Comment:
Remove all this from the ReadMe and leave the heading, at least until we work on it